### DISCUSSION TITLE
can I upload images to paperemerge?

### DISCUSSION THREAD

**Discussion by jalwn** [papermerge/papermerge-core#179] [Q&A] [answered]
<img width="221" alt="image" src="https://github.com/papermerge/papermerge-core/assets/85926950/34eae603-d230-46a2-925a-f0bdc76e745c">

I tired to upload a png file and a jpg file but it wasnt successful. The document gets created but the file archiving does not happen, I cant see any previews, and when I download I cant view it on my computer. I am on a docker compose installation.

**Comment by ciur** [comment, marked as answer]
the jpeg and png files support for 2.1.x is not there. 
The jpeg and png files are supported in 2.0.x though.
The support for jpeg and png files will be added again in 2.2.x.

I recommend you to wait until [end of year 2023](https://github.com/papermerge/papermerge-core/milestone/2) when I will launch 2.2.x version which will, among others,  include support for jpeg and png images.

See issue https://github.com/papermerge/papermerge-core/issues/180

**Reply by jalwn** [reply]
Thank you very much for letting me know!
